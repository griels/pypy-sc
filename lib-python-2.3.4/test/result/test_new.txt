command: /usr/bin/python /u/pedronis/PyPy/dist/pypy/tool/alarm.py 900 /u/pedronis/PyPy/dist/pypy/interpreter/py.py --oldstyle /u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_new.py
run by: pedronis@ratthing-b246
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Thu Apr 28 11:30:41 2005
pypy-revision: 11549
OUTPUT TEST
see output in: /u/pedronis/PyPy/dist/lib-python-2.3.4/test/result/test_new.out
============================================================
faking <type 'unicode'>
faking <type 'module'>
faking <type 'file'>
faking <type 'posix.stat_result'>
faking <type 'posix.statvfs_result'>
Traceback (application-level):
  File "/u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_new.py", line 67 in ?
func = new.function(ccode, g)
TypeError: default __new__ takes no parameters
FAILED: test output differs
========================== closed ==========================
execution time: 5.20506906509 seconds
exit status: 1
