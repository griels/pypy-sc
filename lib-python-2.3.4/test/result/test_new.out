test_new
new.module()
<module object at 0x413ba24c>
new.classobj()
__main__.Spam
new.instance()
<__main__.Spam instance at 0x413c234c>
new.instancemethod()
<method object at 0x413d564c>
new.function()
